src/Chapter23/Listing23.20.DesignatingABlockForUnsafeCode.cs

```diff
@@ -54,7 +54,7 @@ public unsafe static int Main()
         }
     }
 
-    public class VirtualMemoryPtr : System.Runtime.InteropServices.SafeHandle
+    public class VirtualMemoryPtr : System.Runtime.InteropServices.SafeHandle, IDisposable
```

Comment:
I got build errors by removing this because lower down we have this function:
```
        void IDisposable.Dispose()
        {
            // Do not change this code. Put cleanup code in 'Dispose(bool disposing)' method
            Dispose(disposing: true);
            GC.SuppressFinalize(this);
        }
```

Reply:
Correct, I would suggest that we should remove that as well